rust/wal3/src/backoff.rs

```diff
@@ -0,0 +1,55 @@
+use std::collections::hash_map::RandomState;
+use std::hash::BuildHasher;
+use std::time::{Duration, Instant};
+
+//////////////////////////////////////// ExponentialBackoff ////////////////////////////////////////
+
```

Comment:
Please document how this is intended to work (an illustration with all the involved parameter here would be good)

Reply:
I've updated the docs in this module.

Reply:
Nice doc!